project/scripts/unit_tests.py

```diff
@@ -2,28 +2,44 @@
 
 import unittest
 import pandas as pd
+from datetime import datetime, timedelta
 
-from fetch_trends import fetch_hourly_data, aggregate_hourly_to_daily
+from fetch_trends import aggregate_hourly_to_daily
 from dates import get_end_times, get_start_times
 
 
 class TestFetch(unittest.TestCase):
 
     def setUp(self):
-        self.hourly_result = fetch_hourly_data("test", 2021, 1, 1, 2021, 1, 1)
-        self.daily_result = aggregate_hourly_to_daily(self.hourly_result)
+        data = {
+            "test" : [1] * 24
+        }
+        dates = [datetime.now()] * 24
+        hourly_df = pd.DataFrame(data, index=dates)
+        self.daily_result = aggregate_hourly_to_daily(hourly_df)
+
+        longer_data = {
+            "test" : list(range(48))
+        }
+        longer_dates = [datetime.now()] * 48
+        longer_hourly_df = pd.DataFrame(longer_data, index=longer_dates)
+        self.longer_daily_result = aggregate_hourly_to_daily(longer_hourly_df)
```

Comment:
You typically dont store results in setup, that should be specific to test. 

Setting up test data is in setup is perfect as you can pass different combination of this data in multiple test cases, storing results is still done sometimes for convenience, but typically each test case you are testing different things thus results should be different.